### ISSUE TITLE
JetEngine - Repeater field - Woocommerce product variation

### ISSUE THREAD

**Issue by JuliusJensen02** [CrocoBlock/suggestions#6922]
The ability to add a repeater field to woocommerce product variations using JetEngine would be very useful. You can already add normal text fields and so on in the 'Meta Boxes' section, so I can't see why repeater fields should be excluded. 
This would allow people to make product addons with additional data e.g. headings, text and images. 
I know the plugin 'product addons' from woocommerce can do this, but this feature would expand its functionality and I'm sure this feature could be used for many other things as well.

To summarize...
In JetEngine in the section 'Meta Boxes'. When adding meta boxes to 'woocommerce product variations' the ability to add repeater fields is the request.
